pkg/kubeutils/kubeutils.go

```diff
@@ -148,27 +147,26 @@ func WaitForInstallation(ctx context.Context, cli client.Client, writer *spinner
 	return nil
 }
 
-func writeStatusMessage(writer *spinner.MessageWriter, status embeddedclusterv1beta1.InstallationStatus) {
-	if status.State != embeddedclusterv1beta1.InstallationStatePendingChartCreation {
-		writer.Infof("Waiting for additional components to be ready: %s", status.Reason)
+func writeStatusMessage(writer *spinner.MessageWriter, install embeddedclusterv1beta1.Installation) {
+	if install.Status.State != embeddedclusterv1beta1.InstallationStatePendingChartCreation {
 		return
 	}
 
-	chartNames := ""
-	if len(status.PendingCharts) == 0 {
+	if install.Spec.Config == nil || install.Spec.Config.Extensions.Helm == nil {
 		return
-	} else if len(status.PendingCharts) == 1 {
-		// A
-		chartNames = status.PendingCharts[0]
-	} else if len(status.PendingCharts) == 2 {
-		// A and B
-		chartNames = strings.Join(status.PendingCharts, " and ")
-	} else {
-		// A, B, and C
-		chartNames = strings.Join(status.PendingCharts[:len(status.PendingCharts)-1], ", ") + " and " + status.PendingCharts[len(status.PendingCharts)-1]
-	}
-
-	writer.Infof("Waiting for additional components %s to be ready", chartNames)
+	}
+
+	// remove 'admin-console' from this
+	pendingChartsMap := map[string]struct{}{}
+	for _, chartName := range install.Status.PendingCharts {
+		pendingChartsMap[chartName] = struct{}{}
+	}
+	delete(pendingChartsMap, "admin-console")
```

Comment:
instead of deleting the admin-console chart explicitly, how about removing charts that aren't part of the extensions?

Reply:
hmmm
should be doable

Reply:
yeah, then you wouldn't have to check for negatives below and would also handle charts other than the admin-console

Reply:
done